### ISSUE TITLE
Feature request: "Add to collection" function

### ISSUE THREAD

**Issue by zeigerpuppy** [dwrensha/sandstorm-collections-app#7]
It would be really useful to have a function when accessing a grain to add it to a collection.
This would really facilitate work flow rather than having to go to the collection adding process to add a single grain.

I love collections by the way, a really great usability feature!

**Comment by dwrensha** [comment]
I agree. Currently Collections can only pull in grains through powerbox requests. It would be useful if it could also have grains pushed to it, through powerbox offers.
